packages/forms-web-app/src/views/register/organisation/name-of-organisation-or-charity.njk

```diff
@@ -37,6 +37,7 @@
             },
             id: "organisation-name",
             name: "organisation-name",
+						autocomplete: 'organization',
```

Comment:
organi**s**ation is used throughout the code so we should probably keep consistent.